widgetastic_manageiq/__init__.py

```diff
@@ -5195,11 +5195,13 @@ def read_additional_info_popup(self, vm_name):
             el = self.browser.element(self.ITEM_ADDITIONAL_INFO_POPUP_LOCATOR)
             return {
                 "Status": self.browser.text("./h3", parent=el),
-                "Started": self.browser.text("./div[2]/div/div[1]", parent=el),
-                "Description": self.browser.text("./div[2]/div/div[2]", parent=el),
-                "Conversion Host": (self.browser.text("./div[2]/div/div[3]", parent=el)).split(
+                "Started": (self.browser.text("./div[2]/div/div[1]", parent=el)).split(": ")[1],
```

Comment:
Q: Is there any way to get around these indices? This could be difficult to maintain with these pages being updated often.

Reply:
Can I address that in a following PR ?

Reply:
Yep that's fine with me.